packages/compass-collection/src/components/collection-stats-item/collection-stats-item.tsx

```diff
@@ -18,40 +18,41 @@ const collectionStatsItemStyles = css({
 
 const darkThemeLabelStyles = css({
   textTransform: 'uppercase',
-  fontSize: '10px',
-  fontWeight: 'bold',
+  color: uiColors.gray.light1,
   display: 'inline-block',
-  color: uiColors.gray.base,
+  fontWeight: 'bold',
+  fontSize: spacing[2] * 1.5,
+  lineHeight: `${spacing[3]}px`,
 });
 
 const lightThemeLabelStyles = css({
   textTransform: 'uppercase',
-  fontSize: '10px',
-  fontWeight: 'bold',
+  color: uiColors.gray.dark1,
   display: 'inline-block',
-  color: uiColors.gray.dark2,
+  fontSize: spacing[2] * 1.5,
+  lineHeight: `${spacing[3]}px`,
```

Comment:
Is it okay that we use these very custom font sizes here? My understanding was that we stick to what typography packages allows us to use

Reply:
With sizes out of the box, it doesn't match the design 😞

Reply:
Hmmm, shouldn't designs be using spacings from leafygreen though? @Anemy is it some special case here that we decided doesn't need to be aligned with leafygreen?

Reply:
Sorry for missing this, looks like it's gone now so I'll resolve. I think we typically should avoid using `lineHeight` if possible. We should aim to use `spacings` from leafygreen as much as possible. There are rare cases where I think it makes sense to deviate and use custom styles though.